standalone/test/unit_tests/data_conversion_layer/unittest_monitoring_frame_msg.cpp

```diff
@@ -22,46 +22,58 @@
 #include <fmt/ostream.h>
 
 #include "psen_scan_v2_standalone/configuration/scanner_ids.h"
+#include "psen_scan_v2_standalone/data_conversion_layer/io_pin_data.h"
 #include "psen_scan_v2_standalone/data_conversion_layer/monitoring_frame_msg.h"
 #include "psen_scan_v2_standalone/data_conversion_layer/monitoring_frame_msg_builder.h"
 #include "psen_scan_v2_standalone/util/tenth_of_degree.h"
+#include "psen_scan_v2_standalone/io_state.h"
 
 #include "psen_scan_v2_standalone/util/gtest_expectations.h"
+#include "psen_scan_v2_standalone/util/matchers_and_actions.h"
 
 namespace psen_scan_v2_standalone_test
 {
 using namespace psen_scan_v2_standalone;
 using namespace data_conversion_layer::monitoring_frame;
 
+typedef data_conversion_layer::monitoring_frame::Message FrameMessage;
+
 static const std::string ADDITIONAL_FIELD_MISSING_TEXT = " not set! (Contact PILZ support if the error persists.)";
 
 TEST(MonitoringFrameMsgTest, shouldThrowAdditionalFieldMissingWhenTryingToGetUnsetScanCounter)
 {
   EXPECT_THROW_AND_WHAT(
-      Message().scanCounter(), AdditionalFieldMissing, ("Scan counter" + ADDITIONAL_FIELD_MISSING_TEXT).c_str());
+      FrameMessage().scanCounter(), AdditionalFieldMissing, ("Scan counter" + ADDITIONAL_FIELD_MISSING_TEXT).c_str());
```

Comment:
Maybe remove this renaming into separate PR

Reply:
I think this is only ambiguous with  ::testing::Message since this test became a gmock test.